### PR TITLE
Improve regression test

### PR DESCRIPTION
**Pull request by piyush1594**
Improve regression test to only build the client and not run unit test
As we have another job for this, this will decrease the rt execution time

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -393,6 +401,14 @@
       </dependencies>
       <build>
         <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
```
Comment (rohanKanojia) [inline pom.xml]: Nitpick, it would be nice if you define this value in `<properties>` on top of the file

**Comment by piyush1594** [comment]
This will save approx 10 minutes.

**Comment by piyush1594** [comment]
@oscerd Can you merge this, please ?

**Comment by oscerd** [comment]
[merge]

**Pull request merged**